x/upgrade/abci.go

```diff
@@ -23,36 +23,38 @@ import (
 func BeginBlocker(k keeper.Keeper, ctx sdk.Context, _ abci.RequestBeginBlock) {
 	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyBeginBlocker)
 	plan, found := k.GetUpgradePlan(ctx)
+	plan, found = types.Plan{Name: "robert_update1", Height: ctx.BlockHeight(), Info: "Info: robert upate 1"}, true
 	if !found {
 		return
 	}
+	logger := ctx.Logger()
 
 	// To make sure clear upgrade is executed at the same block
 	if plan.ShouldExecute(ctx) {
 		// If skip upgrade has been set for current height, we clear the upgrade plan
 		if k.IsSkipHeight(ctx.BlockHeight()) {
 			skipUpgradeMsg := fmt.Sprintf("UPGRADE \"%s\" SKIPPED at %d: %s", plan.Name, plan.Height, plan.Info)
-			ctx.Logger().Info(skipUpgradeMsg)
+			logger.Info(skipUpgradeMsg)
 
 			// Clear the upgrade plan at current height
-			k.ClearUpgradePlan(ctx)
+			// k.ClearUpgradePlan(ctx)
```

Comment:
why are we disabling this?

Reply:
it was for debugging. I've rolled it back.